patch/llpcPatchInOutImportExport.cpp

```diff
@@ -3182,6 +3199,57 @@ Value* PatchInOutImportExport::PatchFsBuiltInInputImport(
     return pInput;
 }
 
+// =====================================================================================================================
+// Generate code to read BuiltInSamplePosOffset
+Value* PatchInOutImportExport::ReadSamplePosOffset(
```

Comment:
ReadSamplePosOffset -> GetSamplePosOffset

The naming can align with the methods of llpcSystemValues.h

Also, in the file, we already have GetInLocalInvocationId and GetWorkgroupSize